From c73747171c8a91e204405e144d0906c439d3bff3 Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@balena.io>
Date: Mon, 15 Jul 2019 14:05:25 +0100
Subject: [PATCH 16/17] configs: rpi4: Remove DWC2 and USB_ETHER configs

dwc2 is only connected to the usb-c port so we don't have any real
benefit in having it enabled in uboot.

Also, the GENET interface is connected directly to the SoC so we can
drop the USB_ETHER configs.

Signed-off-by: Andrei Gherzan <andrei@balena.io>
---
 configs/rpi_4_32b_defconfig | 3 ---
 configs/rpi_4_defconfig     | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/configs/rpi_4_32b_defconfig b/configs/rpi_4_32b_defconfig
index 9f2b805d0f..b71a14735a 100644
--- a/configs/rpi_4_32b_defconfig
+++ b/configs/rpi_4_32b_defconfig
@@ -31,11 +31,8 @@ CONFIG_PINCTRL=y
 # CONFIG_REQUIRE_SERIAL_CONSOLE is not set
 CONFIG_USB=y
 CONFIG_DM_USB=y
-CONFIG_USB_DWC2=y
 CONFIG_USB_KEYBOARD=y
 CONFIG_USB_HOST_ETHER=y
-CONFIG_USB_ETHER_LAN78XX=y
-CONFIG_USB_ETHER_SMSC95XX=y
 CONFIG_DM_VIDEO=y
 CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_CONSOLE_SCROLL_LINES=10
diff --git a/configs/rpi_4_defconfig b/configs/rpi_4_defconfig
index 83d1bd0cdb..b27e3f823b 100644
--- a/configs/rpi_4_defconfig
+++ b/configs/rpi_4_defconfig
@@ -31,11 +31,8 @@ CONFIG_PINCTRL=y
 # CONFIG_REQUIRE_SERIAL_CONSOLE is not set
 CONFIG_USB=y
 CONFIG_DM_USB=y
-CONFIG_USB_DWC2=y
 CONFIG_USB_KEYBOARD=y
 CONFIG_USB_HOST_ETHER=y
-CONFIG_USB_ETHER_LAN78XX=y
-CONFIG_USB_ETHER_SMSC95XX=y
 CONFIG_DM_VIDEO=y
 CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_CONSOLE_SCROLL_LINES=10
-- 
2.17.1

